docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       - .:/go/src/github.com/datatogether/coverage
       - ./sql:/sql
     ports:
-     - 8080:8080
+      - $PORT:8080
```

Comment:
This change has some consequences that might make it fail to work. The server by default runs on port `8080`, but when `PORT` env var is specified, it would use that port instead (based on a comment in the `config.go` file that says, "// port to listen on, will be read from PORT env variable if present."). Suppose someone sets the `PORT` env var to `80` (or any other value for that matter), this line will be translated to `80:8080`, which means bind port `80` of the host machine to the port `8080` of container. However, nothing is running inside the container on port `8080` because the env var is set to `80` hence the server inside the container is using that instead.